### ISSUE TITLE
Alexa Lists in v4

### ISSUE THREAD

**Issue by NLueg** [jovotech/jovo-framework#1251]
## I'm submitting a...

- [ ] Bug report
- [ ] Feature request
- [x] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior

It should be documented how to ask for permissions to lists and how to add items to the users list.
I actually found a quite old fork where it's documented: https://github.com/gwythyr/jovo-framework-nodejs/blob/master/docs/05_platform-specifics/amazon-alexa/lists.md

## Current Behavior

I have no idea how to include the users list.
I don't found any documentation about this.

## Your Environment

* Jovo Framework version used: 4.2.2

**Comment by jankoenig** [comment]
We don't have convenience classes for this yet, but the permission cards could be added to the response using the [`nativeResponse` property](https://www.jovo.tech/marketplace/platform-alexa/output#native-response).

It might also be helpful to look at the reminders documentation, because the implementation would probably look similar: https://www.jovo.tech/marketplace/platform-alexa/reminders#permissions-consent-card

Happy to take a look at a PR if anyone from the community wants to build this

**Comment by NLueg** [comment]
@jankoenig Thank you for your answer. I will try to implement this by myself and will create a PR for this.
I currently struggle with the Events that Alexa can send to a skill. Not sure, if this is a new issue, but for now I will leave it in a comment.

In Jovo v3 there was an `ON_EVENT` handler (for example like here: https://github.com/jovotech/jovo-templates/blob/7939c654191bc961ecec9b91eac27c29295043e5/alexa/skillevents/typescript/src/app.ts)
I don't found any similar concept for Jovo v4.

**Comment by aswetlow** [comment]
Hey @NLueg 

The `@Handle` decorator is a powerful tool for these use cases. https://www.jovo.tech/docs/handle-decorators 

```typescript
@Handle({
  global: true,
  types: ['AlexaSkillEvent.SkillDisabled'],
  platforms: ['alexa'],
})
handleSkillDisabledEvent() {
  // 
}
```

**Comment by NLueg** [comment]
Ah, okay perfect. I actually looked into the docs, but I think I'm blind 😄 Thank you very much for the fast response!

**Comment by NLueg** [comment]
I actually managed it to implement the functionality in Jovo v4.
Besides the new Handlers I oriented myself heavily on the reminder implementation of the framework.
With this and the documentation of the [Alexa-REST-API](https://developer.amazon.com/en-US/docs/alexa/list-skills/list-management-api-reference.html#get-list-item) it was quite simple.

Thanks again!

**Comment by jankoenig** [comment]
Nice @NLueg! Would you be interested in adding this as a PR? That would be amazing

Here's more information: https://www.jovo.tech/docs/contributing#pr-workflow

**Comment by NLueg** [comment]
@jankoenig sure! Would love to create a PR for this.
You mean for the new API or an example with the required handlers? Or both? 😄

**Comment by jankoenig** [comment]
Great! Whatever you think is helpful for the community, hehe.

Happy to assist if you run into any issues. Thanks!

**Comment by NLueg** [comment]
Can be closed as #1418 was merged 🙂
